src/buttons/CreateScenarioButton/components/CreateScenarioDialog/CreateScenarioDialog.js

```diff
@@ -288,16 +288,17 @@ const CreateScenarioDialog = ({
         </Grid>
       </DialogContent>
       <DialogActions className={classes.dialogActions}>
-        <Button id="cancel" onClick={handleCloseDialog} color="primary">
+        {/* TODO Replace 'cancel' with 'stay connected' which is more accurate */}
```

Comment:
Not sure I see how "stay connected" is related to the scenario creation dialog :sweat_smile:

Reply:
Mauvais endroit... Merci